### PR TITLE
Extend GLAM docs

### PR DESCRIPTION
**Pull request by Iinh**
Provide a short intro to GLAM, since we don't have any documentation for it right now.

### CODE REVIEW

File: src/SUMMARY.md
Comment (wlach) [inline src/SUMMARY.md:8]: We're trying to keep the introduction section pretty clean (I made a pretty major overhaul last summer), so I'd prefer to keep documentation on specific tools either in the cookbook or reference section. If you haven't seen it yet, the style guide has more details about this: https://docs.telemetry.mozilla.org/contributing/style_guide.html

It looks like most of this could be considered a "cookbook". Could you move it to "Tutorials & Cookbooks"? I'd put it right after "Accessing Public Data". IMO GLAM should be one of the first things we want to encourage people with a data-related question to try using.

File: src/tools/glam.md
Comment (wlach) [inline src/tools/glam.md:4]: Can probably skip this.

```suggestion
```

File: src/tools/glam.md
Comment (wlach) [inline src/tools/glam.md:1]:
```suggestion
# Introduction to GLAM
```

File: src/tools/glam.md
Comment (wlach) [inline src/tools/glam.md:7]:
```suggestion
## The differences between GLAM, GUD, and TMO```

File: src/tools/glam.md
Comment (wlach) [inline src/tools/glam.md:15]:
```suggestion
- **[The Telemetry Measurement Dashboard (TMO)](https://telemetry.mozilla.org/)** is the predecessor to GLAM (see above) and is still being lightly maintained until we are sure GLAM covers all of its use cases.
```

I think we now consider GLAM production ready. I'm not sure what the schedule is for taking down TMO: @robhudson does the above sound accurate?

File: src/tools/glam.md
Comment (wlach) [inline src/tools/glam.md:19]:
```suggestion
You can visit GLAM at [glam.telemetry.mozilla.org](https://glam.telemetry.mozilla.org). As of this writing, Mozilla LDAP credentials are required to access it.
```

File: src/tools/glam.md
Comment (wlach) [inline src/tools/glam.md:27]:
```suggestion
GLAM is currently serving data for Firefox Desktop and Firefox for Android. 
```

File: src/tools/glam.md
Comment (wlach) [inline src/tools/glam.md:57]: I'd probably make this section a little more task oriented.

```suggestion
### Going deeper 

If you have a question that can't easily be answered by the GLAM GUI, you can access the raw GLAM datasets using sql.telemetry.mozilla.org.
```

File: src/tools/glam.md
Comment (wlach) [inline src/tools/glam.md:59]:
```suggestion
### Getting help

```

File: src/tools/interfaces.md
Comment (wlach) [inline src/tools/interfaces.md:19]: Can probably take this out now. 😄

```suggestion
```

File: src/tools/interfaces.md
Comment (wlach) [inline src/tools/interfaces.md:17]: I'd probably keep the comparison with GUD on this page, as I think it might be helpful to people. Definitely link out to the more in-depth tutorial too.

File: src/tools/glam.md
Comment (wlach) [inline src/tools/glam.md:23]: We should check these files in locally under `src/assets/` before we land.

File: src/tools/glam.md
Comment (wlach) [inline src/tools/glam.md:60]:
```suggestion
If you have further questions, please reach out on the #glam slack channel.
```

**Review by wlach — CHANGES_REQUESTED** [review]
Hey @Iinh! Thanks for writing this, this is great. I have some initial feedback on where it should live, but I'd say this is almost ready to go. BTW, we generate a "deploy preview" of anything with a submitted pull request to this repo (though hackmd can still be useful for prototyping things IME)

File: src/tools/glam.md
Comment (robhudson) [reply src/tools/glam.md]: Yep that sounds right to me.

File: src/cookbooks/glam.md
Comment (wlach) [inline src/cookbooks/glam.md:53]:
```suggestion
```

File: src/cookbooks/glam.md
Comment (wlach) [inline src/cookbooks/glam.md:56]: We should add a reference to the dataset reference @robhudson is adding in #636 when it lands (can be done in a followup)

```suggestion
If you have a question that can't be easily answered by the GLAM GUI, you can access the raw GLAM datasets using [`sql.telemetry.mozilla.org`](../tools/stmo.md).
```

File: src/cookbooks/glam.md
Comment (wlach) [inline src/cookbooks/glam.md:60]:
```suggestion
If you have further questions, please reach out on the #GLAM slack channel.
```

File: src/cookbooks/glam.md
Comment (wlach) [inline src/cookbooks/glam.md:3]:
```suggestion
GLAM was built to help Mozillians answer their data questions without needing data analysis or coding skills. GLAM contains a visualization layer meant to answer most “easy” questions about how a probe or metric has changed over build ids and releases.
```

File: src/cookbooks/glam.md
Comment (wlach) [inline src/cookbooks/glam.md:13]: Looking at the rendered version (https://deploy-preview-639--firefox-data-docs.netlify.app/cookbooks/glam.html) I feel like this section is a bit out of place in a tutorial and also duplicates the analysis tools article. I'd probably replace it with something like:

```suggestion

GLAM is one of several high-level data tools we provide at Mozilla. For more information, see [Tools for Data Analysis](../tools/interfaces.md).
```

File: src/tools/interfaces.md
Comment (wlach) [inline src/tools/interfaces.md:21]: Can you update the description here with the adjustments you made in the GLAM article (e.g. removing the part about GLAM not being production ready)?

**Review by wlach — APPROVED** [review]
Missed a couple of things the first time around but otherwise looks great! r+wc

File: src/cookbooks/glam.md
Comment (Iinh) [reply src/cookbooks/glam.md:3]: I use `probe/metric` many times in this article, do you think I should replace all such instances?

Comment (wlach) [reply src/cookbooks/glam.md:3]: It's not that big deal, but that seems like a good idea.

File: src/cookbooks/glam.md
Comment (wlach) [inline src/cookbooks/glam.md:54]:
```suggestion
## Going deeper
```

File: src/cookbooks/glam.md
Comment (wlach) [inline src/cookbooks/glam.md:58]: Missed that these were at the wrong level of indentation now, sorry!

```suggestion
## Getting help
```

File: src/cookbooks/glam.md
Comment (wlach) [inline src/cookbooks/glam.md:48]:
```suggestion
If you have a question that can't be easily answered by the GLAM GUI, you can access the raw GLAM datasets using [`sql.telemetry.mozilla.org`](../tools/stmo.md).
```

File: src/cookbooks/glam.md
Comment (wlach) [inline src/cookbooks/glam.md:38]:
```suggestion
- `Percentiles` shows the percentiles of the probe over time. To perform analysis, set a reference point by clicking on a target date, then hover along the graph to see the recorded differences. See attached tooltips on the page for more instruction.
```

**Comment by wlach** [comment]
A few spelling and link nits, other than that I think we're good to go!

**Comment by wlach** [comment]
@Iinh A few more spell check issues it seems. I think you might need to put glam.telemetry.mozilla.org in backticks. I'd recommend running the spell checker locally per the instructions here:

https://github.com/mozilla/data-docs#spell-checking

Sorry for the hassle, I have some thoughts on how to make these kinds of problems less common in the future but that will have to wait until later. 😄

**Review by wlach — APPROVED** [review]
Approved once CI passes

**Pull request merged**
